source/camera_settings/set1/frame_01.yml

```diff
@@ -0,0 +1,23 @@
+__version__: 3
+Settings:
+  Bidirectional: no
```

Comment:
Should these files be located at source/camera_settings/? If they are used just by this one sample, why not place them at  source/Camera/Basic/CaptureHDRLoop/CameraSettings/ ?

Reply:
I though maybe they'd be used by other samples, and also I wanted to separate source code from resources. In fact I would like to discuss maybe having a resources/ folder on the same level as source/. We could then put Zivid3D.zdf there too, and other files until we have a cloud storage...

Reply:
The resources idea seems good. Let's leave the directory structure as it is now.

However, please decide on the directory name so we can make it consistent between different repos.

Reply:
I'm changing the folder name to /settings. I believe it's ok, since the camera settings are placed in subfolders (setX). Thus, it's ok that the name doesn't force <camera>-only settings in it.